index.bs

```diff
@@ -5543,7 +5543,14 @@ This extension enables use of a user verification index.
 
 ## Location Extension (loc) ## {#sctn-location-extension}
 
-This extension provides the [=authenticator=]'s current location to the WebAuthn [=[WRP]=].
+This extension provides the [=authenticator=]'s current geographic location to the [=[WRP]=].
+
+Note: In almost all cases, this information also discloses the location of the user of the device,
+    thereby potentially compromising the user's privacy. A [=conforming user agent=] must provide a mechanism that protects the user's privacy and this mechanism
+    MUST ensure that no location information is made available through this API without the user's
+    express permission [[Permissions]]. Further, the security and privacy considerations
```

Comment:
The location is encoded into the signed authenticator extension output, so it is not possible to mask it out of the response without breaking the signature. The client would have to strip the `loc` extension input from the request to prevent the authenticator to add the output in the first place; maybe we should call that out explicitly.

Reply:
hm, yep, thx, I took a stab at it in 8aaa6ca